recipe/run_test.sh

```diff
@@ -11,10 +11,10 @@
 # https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#test-section
 
 cli() {
-  msg Testing CLI program
+  msg Testing CLI
   (
     set -eux
-    uw --help >/dev/null
+    uw --version
```

Comment:
Now that we have it, we might as well see it when `make test` runs. This also ensures that it doesn't break unexpectedly.